tests/gptq/test_quantization.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tempfile
+import unittest
+
+import torch
+from parameterized import parameterized
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from optimum.gptq import GPTQQuantizer, load_quantized_model
+from optimum.gptq.data import get_dataset
+from optimum.utils.testing_utils import require_accelerate, require_autogptq, require_torch_gpu
+
+
+@require_autogptq
+@require_torch_gpu
+class GTPQTest(unittest.TestCase):
+    model_name = "bigscience/bloom-560m"
+
+    input_text = "Hello my name is"
+    EXPECTED_OUTPUTS = set()
+    EXPECTED_OUTPUTS.add("Hello my name is John and I am a professional photographer. I")
+    EXPECTED_OUTPUTS.add("Hello my name is John.\nI am a friend of your father.\n")
```

Comment:
I'm not sure to get why there's two? In my experience the small numerical difference between runs (due e.g. to unpredictable ordering) should not impact the generation

Reply:
Oh this is something that @younesbelkada did for `bitsandbytes` when we do 4bit quantization. So I just replicated but indeed I've only seen so far the first output with my test. I will only keep the first as you confirmed that it is pretty stable. 
cc @younesbelkada

Reply:
SGTM !